src/index.html

```diff
@@ -4,9 +4,269 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#top-page" class="logo">
+        <img src="./images/logo.png" alt="logo image">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            href="#laptops"
+            data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tab">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="section main__section">
+        <article class="imac-card section__imac-card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac"
+            class="imac-card__image"
+          >
+          <p class="imac-card__name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </p>
+          <p class="imac-card__code">Код товара: 195434</p>
+          <div class="stars stars--rating_4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="imac-card__reviews">Отзывов: 5</p>
+          <p class="imac-card__price-name">Цена:</p>
+          <p class="imac-card__value">69 999 грн</p>
+          <a
+          href="#buy"
+          class="card-action-buy-button"
+          data-qa="card-hover"
+          >
```

Comment:
Keep an eye on indentations.

```suggestion
          <a
            href="#buy"
            class="card-action-buy-button"
            data-qa="card-hover"
          >
```